dissect/target/plugin.py

```diff
@@ -964,4 +1117,4 @@ def find_plugin_functions(
                     )
                 )
 
-    return list(set(result)), invalid_funcs
+    return sorted(list(set(result)), key=lambda key: key.name), invalid_funcs
```

Comment:
What exactly does this sorting add?

Reply:
It solves #266

Reply:
Wouldn't the order still be wrong? The order should be the same as specified on the command line.